File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/LaundryRoom.java
package com.pennapps.labs.pennmobile.classes;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LaundryRoom {

    private int id;
    @SerializedName("hall_name")
    @Expose
    private String name;
    @SerializedName("machines")
    @Expose
    private Machines machines;

    public Machines getMachines() {
        return machines;
    }

    public String getName() {
        return name;
    }

    public int getId() {
        return id;
    }

    public void setId(int newId) {
        id = newId;
    }
}

File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/LaundryRoom.kt
package com.pennapps.labs.pennmobile.classes

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class LaundryRoom {

    var id: Int = 0
    @SerializedName("hall_name")
    @Expose
    val name: String? = null
    @SerializedName("machines")
    @Expose
    val machines: Machines? = null
}
